st2api/tests/unit/controllers/v1/test_auth_api_keys.py

```diff
@@ -13,11 +13,10 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-import httplib
-
 import mock
 from oslo_config import cfg
 
+from six.moves import http_client
```

Comment:
Please use consistent import order:

1. System level imports
2. External imports (such as six)
3. Local project imports

Each of the section separated by a blank line.